ppr-ui/src/utils/axios-pay.ts

```diff
@@ -0,0 +1,28 @@
+import Axios from 'axios'
+import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
+import * as Sentry from '@sentry/browser'
+
+const axios = Axios.create()
+
+axios.interceptors.request.use(
```

Comment:
what is this used for? just a quick comment would help :)

Reply:
sure, I'll add a quick explanation